data/json/items/book/computer.json

```diff
@@ -111,10 +111,52 @@
     "looks_like": "textbook_traps",
     "color": "blue",
     "skill": "computer",
-    "required_level": 3,
+    "required_level": 4,
     "max_level": 6,
     "intelligence": 10,
     "time": "35 m",
     "fun": -2
+  },
+  {
+    "id": "webbasics_computer",
+    "type": "BOOK",
+    "name": { "str": "Web Design Basics", "str_pl": "copies of Web Design Basics" },
+    "//": "A thick glossy paperback instructional manual for aspiring web designers. Includes a teaching guide for young students.",
+    "description": "An entry-level book for younger coders that includes real coding practice.",
+    "weight": "907 g",
+    "volume": "500 ml",
+    "price": 2000,
+    "price_postapoc": 50,
+    "material": [ "paper" ],
+    "symbol": "?",
+    "looks_like": "textbook_traps",
+    "color": "green",
+    "skill": "computer",
+    "required_level": 1,
+    "max_level": 3,
+    "intelligence": 7,
+    "time": "20 m",
+    "fun": -1
+  },
+  {
+    "id": "hackerman_computer",
+    "type": "BOOK",
+    "name": { "str": "Pro Hacker Secrets", "str_pl": "copies of Pro Hacker Secrets" },
+    "//": "A thinly veiled attempt at teaching kids coding skills. Not nearly as exciting as the authors might have hoped, but there's some real knowledge in here.",
```

Comment:
I think some or all of the descriptive text you have added to the comments for these books would be suitable in the description proper - at least in this case, I prefer the more colorful language of the comment.

Reply:
Great! I think I originally meant to do it the other way around, but this worked as a way of seeing if people approved of the language too.